search-api.md

```diff
@@ -23,12 +23,6 @@ After receiving a request, the remote server can respond in one of two ways:
   * If a compatible version (`vX.Z` where `Z>=Y`) is supported by the remote server, it should provide a response using this version.
   * If no appropriate version is supported by the remote server, it should respond with `Not Acceptable (406)`, containing a JSON body with a description of the error. All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server. This will enable the user to re-submit the request using this version of the API.
```

Comment:
What @allasm is complaining about is this part: 

> All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server.

This does conflict a bit with the previous line:

> If a compatible version [...] is supported by the remote server, it should provide a response using this version.

How do we define _compatible_? Are bugfix `X.*` versions supposed to be compatible, with a clearly defined measure of [in]compatibility? Should the exact same version as in the request be used, or do we consider that all versions sharing the same major number are compatible and a client supporting one version should be able to handle (possibly ignoring certain fields) any other version from the same branch?